components/QuickLinks.vue

```diff
@@ -54,22 +53,22 @@ export default {
           link: 'https://demo.arcana.network',
           linkText: 'View App',
         },
+        {
+          name: 'dashboard',
+          title: 'Go to Dashboard',
+          link: 'https://dashboard.arcana.network',
+          linkText: 'Go to Dashboard',
+        },
         {
           name: 'discord',
-          title: 'Get Support on Integration',
+          title: 'Get Integration Support',
           link: 'https://discord.gg/w6ej4FtqYS',
           linkText: 'Join Discord',
         },
         {
-          name: 'signup',
-          title: 'Get Started with our SDK',
-          link: 'https://dashboard.arcana.network',
-          linkText: 'Sign up',
-        },
-        {
-          name: 'discourse',
+          name: 'report-bugs',
           title: 'Report Bugs',
-          link: 'https://forum.arcana.network',
+          link: '#',
```

Comment:
We'll still be using `https://forum.arcana.network/` for bug reports, I believe. Can you confirm with Anirudh?

Reply:
Yeah, I'll confirm this

Reply:
have you confirmed this? @shrinathprabhu